tests/requests/tests.py

```diff
@@ -743,6 +743,135 @@ def test_get_host_suggestion_of_allowed_host(self):
         )
 
 
+class DataUploadMaxMemorySizeGetTests(SimpleTestCase):
+
+    def _test_data_upload_max_memory_size_exceeded(self, content_length_header):
+        request = WSGIRequest({
+            'REQUEST_METHOD': 'GET',
+            'wsgi.input': BytesIO(b''),
+            content_length_header: 3,
+        })
+
+        with self.settings(DATA_UPLOAD_MAX_MEMORY_SIZE=2):
+            with self.assertRaisesMessage(SuspiciousOperation, 'Request data too large'):
+                request.body
+
+        # no problem for body <= DATA_UPLOAD_MAX_MEMORY_SIZE
+        with self.settings(DATA_UPLOAD_MAX_MEMORY_SIZE=3):
+            request.body
+
+        # After a successful read, attributes on request are cached so need to
+        # create a new one.
+        request = WSGIRequest({
+            'REQUEST_METHOD': 'GET',
+            'wsgi.input': BytesIO(b''),
+            content_length_header: 3,
+        })
+
+        # ... or DATA_UPLOAD_MAX_MEMORY_SIZE=None
```

Comment:
rephrase this comment (it has ... in the beginning, but does not continue the previous comment, but the one before that, which is unclear).